articles/Rules/Content_usage_guidelines/en.md

```diff
@@ -0,0 +1,47 @@
+# Content usage guidelines
+
+Kumi as a rhythm-based game is an experience that is meant to be shared with others; through means of charting; and with the use of the game's built-in editor, it is possible to create and share your own charts with others. However, there are some guidelines that must be followed when creating and sharing charts; such as preserving the integrity of a creator's work, and attributing the proper credit to the original creators of the content used in the derivative works that are created.
+
+Additionally, there are guidelines for the visual content that is used in the game; such as the use of custom skins, and the use of custom backgrounds; and
+ensuring that the content used in the game is appropriate for all audiences.
+
+## Gameplay
+
+When creating or submitting charts, any users involved in the creation of the chart should have the necessary permissions to use the content that is used in the chart, including charts that are created for other games. We do not condone the use of gameplay content that is not owned by the creator of the chart, or the use of content that is not licensed for use in other derivative works.
+
+This includes, but is not limited to:
+* The use of charts from other rhythm games not owned by the creator of the chart;
+* The use of any difficulties from other charts that are not owned by the creator of the chart;
+* And any gameplay content used in a chart that is stripped from another game.
+
+This is to ensure that the integrity of the original creator's work is preserved, and that the original creator is properly credited for their work; and to
```

Comment:
```Suggestion
This is to ensure that the integrity of the original creator's work is preserved, that the original creator is properly credited for their work, and to ensure that the content in Kumi is unique to Kumi itself unless otherwise stated.
```